IanGRap/Squish
# Hide Mouse Cursor

Contributors: Austin

Goal: 
- Hide the mouse cursor in standalone build

What changed:
- Added a function to GameSettingsManager that hides the mouse cursor if not inside the editor

How to Test:
- Load up any scene.
- Make sure you can still see the mouse cursor in the Unity editor
- Make a build and ensure that once the title screen loads up, the cursor is not displayed
